src/Button/Button.js

```diff
@@ -156,6 +159,7 @@ const styles = {
         text === undefined
           ? theme[`Button_paddingIconOnly_${size}`]
           : `0 ${theme.Button_paddingHorizontal}`,
+      textDecoration: 'none',
```

Comment:
We may want to consider some additional preventative styles:

- Add `textDecoration: 'none'` to `&:hover` and `&:focus`
- Prevent the `color` from changing on hover/focus/active

Reply:
hot